.github/workflows/update_client.yaml

```diff
@@ -0,0 +1,95 @@
+name: Update SDK
+
+# Define the events that trigger the workflow
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "api/**"
+  pull_request:
+    types: [closed]
+    paths:
+      - "api/**"
```

Comment:
Why do we need to build the SDK when a PR is closed? A closed PR is either merged to _main_ (which falls in the previous case) or to _develop_ which I think we're not interested in using to build an SDK?